Dashboard button soft-launch info updated (#3952)
SkylineCommunications/dataminer-docs: user-guide/Reference/Soft-launch_options/Overview_of_Soft_Launch_Options.md

### ReportsAndDashboardsAutomationScript

> Enables the Automation script component in the Dashboards app, which is visualized as a button. This component can be used to execute scripts. It requires no data but can have elements or parameters as filters. If the *ReportsAndDashboardsButton* option is also enabled, Automation script settings are enabled for the button component in the *Settings* pane, which allows the component to be used to execute scripts.

Reviewer: The 'ReportsAndDashboardsButton' flag doesn't change any behavior until you add a button parameter. The Automation script settings are not linked to this flag, but to the 'ReportsAndDashboardsAutomationScript' flag

Author: ```suggestion
Enables the Automation script component in the Dashboards app, which is visualized as a button. This component can be used to execute scripts. It requires no data but can have elements or parameters as filters. If the *ReportsAndDashboardsButton* option is also enabled, one button component will be available that can be used with an Automation script or with a button parameter.
```
